src/cmdict/run_script.py

```diff
@@ -160,6 +179,8 @@ def _echo_item(word, res):
             + Style.RESET_ALL
             + " can not be found in the database!"
         )
+        _found = False
```

Comment:
Why do you need this?

Reply:
Initially planned to store those found in the database only.

Reply:
That part doesn't have an impact right now.

Reply:
No, I pretty much against it unless you have a very strong reason. A function should only do what it supposes to do. i.e. `_echo_item` should only "echo items". If you want to check if a word exist or not, you can add a method in the connector.

Reply:
"Initially planned to store those found in the database only." would be implemented in another function, not in `_echo_item `. For now, I just enable `_echo_item ` to return information on whether it find the word or not, in case we need this information in the future.

Reply:
If you want to check if a word exist or not, you can add a method in the connector.